.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test-score-and-rank-contributors"]
+	path = test-score-and-rank-contributors
+	url = https://github.com/anminliu/test-score-and-rank-contributors.git
```

Comment:
The repo name is a bit misleading. The name seems to suggest that the repo tests the tool. But this is not the case. It is only used for testing purpose. So let's give it a more appropriate name. How about `alice-and-lam`?

```markdown
# Alice and Lam

This Git repository is collaborative work by two fictive developers named Alice and Lam.
The repository's content is not meaningful,
its whole purpose is to provide a deterministic commit history
so that the [`score-and-rank-contributors`](https://github.com/asterinas/score-and-rank-contributors) tool
can use it for the testing purpose.
```

To further convey the intention that the content in the repo is meaningless, use [Lorem ipsum](https://en.wikipedia.org/wiki/Lorem_ipsum) for the content of any file in the repo (except the README).

After fixing the name and content of the test repo, transfer its ownership to asterinas. And use `https://github.com/asterinas/alice-and-lam.git` as the URL.